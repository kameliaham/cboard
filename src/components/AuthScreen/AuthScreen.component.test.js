import React from 'react';
import { shallow, mount } from 'enzyme';
import { shallowMatchSnapshot } from '../../common/test_utils';
import AuthScreen from './AuthScreen.component';

jest.mock('./AuthScreen.messages', () => {
  return {
    login: {
      id: 'cboard.components.WelcomeScreen.login',
      defaultMessage: 'Login'
    },
    signUp: {
      id: 'cboard.components.WelcomeScreen.signUp',
      defaultMessage: 'Sign Up'
    },
    facebook: {
      id: 'cboard.components.WelcomeScreen.facebook',
      defaultMessage: 'Sign in with Facebook'
    },
    google: {
      id: 'cboard.components.WelcomeScreen.google',
      defaultMessage: 'Sign in with Google'
    },
    skipForNow: {
      id: 'cboard.components.WelcomeScreen.skipForNow',
      defaultMessage: 'Skip for now'
    }
  };
});

describe('AuthScreen tests', () => {
  test('default renderer', () => {
    shallowMatchSnapshot(<AuthScreen history={{ goBack: () => {} }} />);
  });
});
