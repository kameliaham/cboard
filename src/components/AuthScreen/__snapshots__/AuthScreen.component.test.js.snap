// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthScreen tests default renderer 1`] = `
<AuthScreen
  history={
    Object {
      "goBack": [Function],
    }
  }
  intl={
    Object {
      "formatMessage": [Function],
    }
  }
/>
`;
